raiden/utils/__init__.py

```diff
@@ -261,3 +261,13 @@ def optional_address_to_string(address: typing.Address = None) -> typing.Optiona
         return None
 
     return to_checksum_address(address)
+
+
+def safe_gas_limit(*estimates) -> int:
+    """ Calculates a safe gas limit for a number of estimates.
+
+    Even though it's not documented, it does happen that estimate_gas returns `None`.
```

Comment:
This won't make sense anymore once https://github.com/raiden-network/raiden/pull/2988 is merged, perhaps we should remove it?

Reply:
I wonder if only returns `None` os failure scenarios.